meinberlin/config/settings/base.py

```diff
@@ -323,7 +327,11 @@
             ['Image'],
             ['NumberedList', 'BulletedList'],
             ['Link', 'Unlink'],
+            ['Embed', 'EmbedBase']
         ],
+        'removePlugins': 'stylesheetparser',
+        'extraAllowedContent': 'iframe[*]',
+        'extraPlugins': ','.join(['embed', 'embedbase']),
     },
```

Comment:
was this from the old commit? I wonder did you try only adding it to the `collapsible-image-editor` ?